packages/gamut-labs/src/GlobalHeader/GlobalHeaderItems.tsx

```diff
@@ -228,6 +229,21 @@ const profileBusinessAccount: AppHeaderLinkItem = {
   type: 'link',
 };
 
+let authorUrl = `https://author.codecademy.com`;
+if (process.env.NODE_ENV === 'staging') {
```

Comment:
what about checking that i's not the production environment so that zoo servers can connect to author staging
```suggestion
if (process.env.NODE_ENV !== 'production') {
```

Reply:
I think it doesn't really matter since you won't be able to follow data anyway

Reply:
but maybe going to prod on a test env is bad ill switch